doc/src/manual/strings.md

```diff
@@ -491,61 +396,40 @@ julia> length.([a, b, c])
  1
 ```
 
-This situation can happen only for invalid UTF-8 strings. For valid UTF-8 strings
-concatenation preserves all characters in strings and additivity of string lengths.
+この状況は，無効な UTF-8 文字列に対してのみ発生します．有効な UTF-8 文字列の場合，連結は文字列内のすべての文字と文字列長の加法性を保持します．
 
-Julia also provides [`*`](@ref) for string concatenation:
+また，Julia には文字列連結のための[`*`](@ref)が用意されています:
 
 ```jldoctest stringconcat
 julia> greet * ", " * whom * ".\n"
 "Hello, world.\n"
 ```
 
-While `*` may seem like a surprising choice to users of languages that provide `+` for string
-concatenation, this use of `*` has precedent in mathematics, particularly in abstract algebra.
+一方，文字列の連結に `+` を提供している言語のユーザーにとっては，`*` は意外な選択に思えるかもしれませんが，`*` の使用は，数学，特に抽象代数では前例があります．数学では，`+`は通常，被演算子の順序が問題にならない _可換_ の演算を表します．この例として行列の加算を考えると，同じ形の行列 `A` と `B` に対して`A + B == B + A` となります．対照的に，`*` は一般的に _非可換_ 演算を表し，演算子の順序が重要になります．この例として行列の乗算を考えると，一般的には `A _ B != B _ A` となります．行列の乗算と同様に文字列の連結も _非可換_ です．例えば，`greet _ whom != whom _ greet`となります．このように， 中置記法の文字列連結演算子としては `*` がより自然な選択であり，一般的な数学的使用と一致しています．
```

Comment:
Please replace `A _ B != B _ A` to `A * B != B * A`, and also `greet _ whom != whom _ greet` to `greet * whom != whom * greet`.

Reply:
@alicegawa  Thank you for pointing it out! I fix it!

Reply:
@jabelic Thank you for your quick response, I'll merge it!